tests/WebAssertTest.php

```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExistsValid()
+    {
+        $page = $this->getMockBuilder('Behat\\Mink\\Element\\DocumentElement')
+                     ->disableOriginalConstructor()
+                     ->getMock()
+        ;
+
+        $element = $this->getMockBuilder('Behat\\Mink\\Element\\NodeElement')
+                        ->disableOriginalConstructor()
+                        ->getMock()
+        ;
+
+        $this->session
+            ->expects($this->exactly(1))
+            ->method('getPage')
+            ->will($this->returnValue($page))
+        ;
+
+        $page
+            ->expects($this->exactly(1))
+            ->method('find')
+            ->with('css', 'h2 > span')
+            ->will($this->returnValue($element))
+        ;
+
+        $element
+            ->expects($this->exactly(1))
+            ->method('doesNotHaveAttribute')
+            ->with('blah')
+            ->will($this->returnValue(true))
+        ;
+
+        call_user_func_array(array($this->assert, 'elementAttributeNotExists'), array('css', 'h2 > span', 'blah'));
```

Comment:
btw, it would be better to sue the existing ``assertCorrectAssertion`` and ``assertWrongAssertion`` methods instead